libr/bin/format/mach0/mach0.c

```diff
@@ -167,8 +182,8 @@ static bool init_hdr(struct MACH0_(obj_t) *bin) {
 	bin->hdr.reserved = r_read_ble (&machohdrbytes[28], bin->big_endian, 32);
 #endif
 	sdb_set (bin->kv, "mach0_header.format",
-		"xxxxddx "
-		"magic cputype cpusubtype filetype ncmds sizeofcmds flags", 0);
+		"xxx[4]Edd[4]B "
```

Comment:
Don't forget to add also all the new mach0 stuff like "mach0_dylib_command" that you added in libr/bin/d/macho again here :p

Reply:
yes, i'll do that. it's very unfortunate that we have to dupe things here

Reply:
Yes, It's very problematic because for it's full duplicate and a pain to maintain. 

I proposed multiple time that we should just load those depending on the file format/arch just like we do for types/functions signatures etc.